test/onegini_test.dart

```diff
@@ -1 +1,4 @@
+// TODO implement tests or remove file
+void main() {
```

Comment:
🤔

Reply:
temp to fix tests, will be further looked at during test rework

Reply:
If you have a separate ticket in order to solve that, please add jira link in the comment 👍

Reply:
During the dagger injection ticket we will mostly rework the existing tests and then I will also take a second look at these things https://onewelcome.atlassian.net/browse/FP-31